aries-site/src/pages/components/pageheader.mdx

```diff
@@ -0,0 +1,244 @@
+
+The region containing the title and subtitle should:
+
+- Flex to fill the available horizontal space on the page.
+- Have a minimum width of "medium" on `medium-xlarge` breakpoints and "small" on `xsmall-small` breakpoints to avoid excessive wrapping.
+
+Page-level actions should:
+
+- Have a width of "auto" by default.
+- Progressively collapse into an overflow menu as screen width becomes limited. This should follow guidance on [responsive behavior of actions](#responsive-behavior-of-actions).
+
+<Notification
+  width="large"
+  status="info"
+  message="Resize your window to see how the content regions adapt at various screen widths."
+/>
+
+<Example showResponsiveControls={[]} screenContainer plain width="100%">
+  <PageHeaderContentRegions background="background-front" />
+</Example>
+
+## Title
+
+A title is a required element of the page and should:
+
+- Always be an h1 with `size="small"`.
+- Uniquely and concisely identify the page. No two pages under the same parent should have the same name.
+- Follow title case capitalization (i.e., List of Clusters, My Account).
+- Wrap if too long for the available space. See [Responsive Behavior and Content Prioritization](#responsive-behavior-and-content-prioritization) for more guidance.
+- Be left-aligned with page content.
+
+## Subtitle
+
+A subtitle is an optional element that can provide additional contextual information about the contents of the page.
+
+A subtitle should:
+
+- Concisely elaborate what the page is on. This could be additional information such as ownership of an application, timestamps, or a name to represent by whom a piece of information was created.
+- Be located below the title and left-aligned with page content.
+- Wrap at the max-width for the paragraph's font-size in order to optimize the readability.
+
+<BestPracticeGroup columns="100%">
+  <Example
+    bestPractice={{
+      type: 'do',
+      message:
+        'Include concise, high-priority metadata in the subtitle that aids page identification.',
+    }}
+    width="100%"
+    height={{ min: 'small' }}
+  >
+    <PageHeaderSubtitle />
+  </Example>
+  <Example
+    bestPractice={{
+      type: 'dont',
+      message: `Don't clutter the Page Header with all of the page metadata. Instead, include this in the page content.`,
+    }}
+    width="100%"
+    height={{ min: 'small' }}
+  >
+    <PageHeaderSubtitle bestPractice={false} />
+  </Example>
+</BestPracticeGroup>
+
+## Reverse Anchor
+
+A reverse Anchor is required on all child pages and should:
+
+- Be located above the title and left-aligned with page content.
+- Be labelled with the name of the parent page and accompanied by the "FormPrevious" icon.
+- Not be used on [top-level pages](#example-of-top-level-page) because they do not have a parent.
```

Comment:
```suggestion
- Navigate the user from the child page to its parent.
- Not be used on [top-level pages](#example-of-top-level-page) because they do not have a parent.
```